Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -552,7 +628,26 @@ private IEnumerator CheckButtonTranslation(Vector3 targetStartPosition, Transfor
                 wasTranslated |= targetStartPosition != translateTarget.localPosition;
             }
 
-            Assert.True(wasTranslated, "Transform target object was not translated by action.");
+            Assert.AreEqual(shouldTranslate, wasTranslated, "Transform target object did or did not translate properly by action.");
+        }
+
+        private IEnumerator RunGlobalClick(IMixedRealityInputSource defaultInputSource, 
+            MixedRealityInputAction inputAction, 
+            Vector3 targetStartPosition, 
+            Transform translateTargetObject, 
+            bool shouldTranslate = true)
+        {
+            // Raise a select down input event, then wait for transition to take place
+            // Wait for at least one frame explicitly to ensure the input goes through
+            CoreServices.InputSystem.RaiseOnInputDown(defaultInputSource, Handedness.Right, inputAction);
+            yield return new WaitForFixedUpdate();
```

Comment:
This was copied and pasted from original code. 

I can convert to input system update waiting..but then that begs should all tests be updated in that manner? Because this one was left out as the old way

Reply:
If you check in with WaitForFixedUpdate here, you will very likely add an intermittent test failure, as I just fixed that a few days ago. I think you copied the code before my fix. We can create a bug to track fixing everything, but for now I'd say "if it improves the code and doesn't regress stuff, and follows our principles, it's good". 

But please don't re-add this waitforfixedupdate, it took me a while to figure out and fix the bug. You can test this yourself by bulk running the tests in sequence.

Reply:
It's already been fixed :)

I am actually seeing intermittent failures on InputAction locally with the stabilization branch...so It'll probably be good to have the WaitForInputSystem for all of InteractableTests